pages/doc/proxies_configuring.md

```diff
@@ -453,10 +458,12 @@ Ex: 2978,2979</td>
 <td>&nbsp;</td></tr>
 <tr>
 <td>retryThreads</td>
-<td>Number of threads retrying failed transmissions. If no value is specified, defaults to the number of processor cores available to the host or 4, whichever is greater. Every retry thread uses a separate buffer file (capped at 2GB) to persist queued data points, so the number of threads controls the maximum amount of space that the proxy can use to buffer points locally.</td>
+<td>Number of threads retrying failed transmissions. If no value is specified, defaults to the number of processor cores available to the host or 4, whichever is greater. Every retry thread uses a separate buffer file (capped at 2GB) to persist queued data points, so the number of threads controls the maximum amount of space that the proxy can use to buffer points locally.
+{% include warning.html content="This configuration was deprecated in version 6.0 because we redesigned and improved the storage engine to resolve the problem of spooling data to disk." %}
```

Comment:
because we redesigned and improved the storage engine to resolve the problem of spooling data to disk.

=> because we redesigned the storage engine to improve spooling data to disk.

(spooling data isn't a problem, is it? )